mindmeld/cli.py

```diff
@@ -785,6 +792,36 @@ def _get_auto_annotator_config(app_path, overwrite=False, unannotate_all=False):
     return config
 
 
+@shared_cli.command("augment", context_settings=CONTEXT_SETTINGS)
+@click.option(
+    "--app-path",
+    required=True,
+    help="The application's path.",
+)
+@click.option(
+    "--lang",
```

Comment:
I would advice to use the flag name `language` here to keep it consistent with existing api terminology: https://www.mindmeld.com/docs/userguide/internationalization.html?highlight=language%20code

Reply:
updated!